### ISSUE TITLE
Add an API for external data usage

### ISSUE THREAD

**Issue by adamculp** [php-ug/php.ug#184]
Would be nice if there was an API to allow external usage of this data. Perhaps offer external feeds for XML/RSS and JSON.

**Comment by heiglandreas** [comment]
You mean something like http://php.ug/api/rest/listtype.json/1 ?

**Comment by heiglandreas** [comment]
You mean something like http://php.ug/api/rest/listtype.json/1? I don't see the benefit in adding an RSS-Feed but I'm sure you can enlighten me there ;)

**Comment by heiglandreas** [comment]
There's also the possibility to subscribe to a calendar with events around your area. This one f.i. is for Usergroup-events around Frankfurt (216km radius): webcal://php.ug/api/v1/calendar/list?format=ical&location=50.14874640066278+8.525390625&latitude=50.14874640066278&longitude=8.525390625&distance=216

**Comment by adamculp** [comment]
Yes, that is kinda what I was thinking. This enhancement came to mind when I saw some others creating a JSON based UG list "that can be open for everyone to use".

How extensive is this API? I see no mention of it anywhere on the blog or as a link on the map pages. Maybe this ticket should be updated to "add docs and developer links to the API"?

**Comment by heiglandreas** [comment]
BTW: There's now also an API-Documentation at https://php-ug.github.io/php.ug/api/
